### PR TITLE
Change user-agent for Analytics HTTP requests

### PR DESCRIPTION
**Pull request by chanseokoh**
Turns out that we have a great degree of flexibility in choosing a user-agent string.

### CODE REVIEW

**Comment by etanshaul** [comment]
Do you mean it can be set to anything and we still get the same effect?

**Comment by chanseokoh** [comment]
No, the previous string `"Apache-HttpClient/4.4.1 (Java/1.7.0_101)"` was problematic. I checked that this one in the PR works. Eclipse will use its own string as well.

File: google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java [hunk 1]
```diff
@@ -111,10 +114,10 @@ public FluentTrackingEventWithLabel trackEvent(String action) {
     return new TrackingEventBuilder(this, externalPluginName, action);
   }
 
-  private static void sendPing(@NotNull final List<? extends NameValuePair> postData) {
+  private void sendPing(@NotNull final List<? extends NameValuePair> postData) {
```
Comment (etanshaul) [inline google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java]: you made this non-static but I don't see any change to a caller of this?

Comment (chanseokoh) [inline google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java]: It's to use `userAgent`. Another way could be making `userAgent` static (and possibly other fields). Which one do you think is better?

**Comment by etanshaul** [comment]
Oh I see. it looks like sendPing is only called from a non-static context so what you did looks fine.

**Comment by etanshaul** [comment]
lgtm

**Pull request merged**